cmake/util/FindVulkan.cmake

```diff
@@ -60,7 +60,7 @@ macro(find_vulkan use_vulkan)
    # additional libraries
 
   if(Vulkan_FOUND)
-    get_filename_component(VULKAN_LIBRARY_PATH ${Vulkan_LIBRARY} DIRECTORY)
+    set(VULKAN_LIBRARY_PATH ${__vulkan_sdk}/lib)
```

Comment:
We cannot use `__vulkan_sdk` here directly, because it may not be available in the alternative case (Cmake's official find vulkan)

Reply:
Update to add extra hint.